hw-1/jmp-cloud-bank-impl/pom.xml

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <parent>
+        <artifactId>hw-1</artifactId>
+        <groupId>com.epam</groupId>
+        <version>1.0-SNAPSHOT</version>
+    </parent>
+    <modelVersion>4.0.0</modelVersion>
+
+    <artifactId>jmp.cloud.bank.impl</artifactId>
+    <dependencies>
+        <dependency>
+            <groupId>com.epam</groupId>
+            <artifactId>jmp.bank.api</artifactId>
+            <version>1.0-SNAPSHOT</version>
+            <scope>compile</scope>
```

Comment:
compile is default scope. It is redundant to specify that explicitly.

Optional: delete scope.